src/main/java/com/conveyal/datatools/common/status/MonitorableJob.java

```diff
@@ -181,7 +181,9 @@ public void run () {
             // TODO: should we add separate hooks depending on state of job/sub-tasks (e.g., success, catch, finally)
             // Set status to finished in case duration needed by finishing stage. (e.g., storing the job duration in a
             // database).
-            status.completeSuccessfully("Job complete!");
+            if (cancelMessage.equals("")) {
+                status.completeSuccessfully("Job complete!");
+            }
```

Comment:
Could you update the comment for this conditional? Why does an empty string indicate success... I feel like this would be a little more legible if you were checking that either the subtask or parent task errored.

Reply:
I'm going to go ahead and implement this fix.